templates/mongodb-atlas.base-template.yaml

```
@@ -0,0 +1,329 @@
+---
+AWSTemplateFormatVersion: '2010-09-09'
+Description: MongoDB Atlas AWS CloudFormation Quickstart. (qs-1rkorhec7)
```

Can't reuse template IDs. Replace with: `qs-1tq8f0hrl`.
```suggestion
Description: MongoDB Atlas AWS Partner Solution. (qs-1tq8f0hrl))
```